ui/codegen.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
This is much easier to read than the long string in the `package.json`. Great improvement.

Not for this PR, but I wonder if we should add the underlay config typescript generation here too.
`./gradlew annotationProcessor:run -Pgenerator=UNDERLAY_CONFIG_TYPESCRIPT`
Currently, we're generating the ts files and checking them into GH, but seems like it would be better to generate at build and gitignore them.

Reply:
Or alternatively, maybe all of this code gen should be run from gradle so there's a single place for it. The only use case I can think of for keeping them separate is some small subset of unit tests that use the generated code but also don't need the backend.